### PR TITLE
named schema for requestbody

### PR DESCRIPTION
**Pull request by nomonamo**
This PR addresses the issue described in #47 🤓

### CODE REVIEW

**Review by wI2L — COMMENTED** [review]
LGTM overall.

File: openapi/generator.go [hunk 1]
```diff
@@ -408,6 +408,26 @@ func (g *Generator) setOperationParams(op *Operation, t, parent reflect.Type, al
...
+		mt := tonic.MediaType()
+		if mt == "" {
+			mt = anyMediaType
+		}
+		sch := op.RequestBody.Content[mt].Schema
+		if sch != nil {
+			name := strings.Title(op.ID) + "Input"
+			g.api.Components.Schemas[name] = sch
+			op.RequestBody.Content[mt].Schema = &SchemaOrRef{Reference: &Reference{
+				Ref: "#/components/schemas/" + name,
```
[the remark this answers is not in the document]
Comment (nomonamo) [reply openapi/generator.go:427]: Absolutely, let me refactor that 👍

**Review by wI2L — APPROVED** [review]

**Comment by wI2L** [comment]
@nomonamo Can you squash and sign your commit ?

**Review by wI2L — APPROVED** [review]

**Comment by wI2L** [comment]
Thanks for the feature. 🚲

**Pull request merged**